.ci/teardown.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+#
+# Copyright (c) 2017-2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+log_copy_dest="$1"
+
+runtime_log_filename="kata-runtime.log"
+runtime_log_path="${log_copy_dest}/${runtime_log_filename}"
+runtime_log_prefix="kata-runtime_"
+
+proxy_log_filename="kata-proxy.log"
+proxy_log_path="${log_copy_dest}/${proxy_log_filename}"
+proxy_log_prefix="kata-proxy_"
+
+shim_log_filename="kata-shim.log"
+shim_log_path="${log_copy_dest}/${shim_log_filename}"
+shim_log_prefix="kata-shim_"
+
+crio_log_filename="crio.log"
```

Comment:
wrt the commit message - it also appears to collect from CRIO and docker - do you want to note that as well?

Reply:
ohh right , will update with that info :)